song_player/ui_songplayer.h

```diff
@@ -0,0 +1,69 @@
+/********************************************************************************
```

Comment:
I guess you have to commit this out first before adding it to the `.gitignore` :p